src/test/javascript/spec/service/string-count.service.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { StringCountService } from 'app/exercises/text/participate/string-count.service';
+
+describe('WordCountService', () => {
+    let service: StringCountService;
+
+    const TEXT_WITH_63_WORDS =
+        'Deutsches Ipsum Dolor quo Grimms Märchen posidonium Kaftfahrzeug-Haftpflichtversicherung adhuc Schnaps sadipscing Krankenschwester at, ' +
+        'Sprechen Sie deutsch mei Handschuh gloriatur. Freude schöner Götterfunken inermis Apfelstrudel accommodare Berlin Id Angela Merkel assum Fernweh te ' +
+        'Fernweh erroribus bitte Nec Bildung amet Guten Tag iriure, Joachim Löw gloriatur Aperol Spritz ut. Brezel virtute Goethe per Schweinsteiger ' +
+        'At 99 Luftballons sßaevola Weltanschauung An Hochzeit malorum Prost ius';
+
+    const EMPTY_TEXT = '';
+
+    beforeEach(() => {
+        service = new StringCountService();
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
+
+    it('German text should contain 63 words', () => {
```

Comment:
"German" text :smile:

Reply:
:D Changed it in c33ca84b7c44108ace0276aa50fe3ea98fc3d4f8